src/__tests__/patients/related-persons/RelatedPersonsTab.test.tsx

```diff
@@ -77,10 +77,9 @@ describe('Related Persons Tab', () => {
     })
 
     it('should not render a New Related Person button if the user does not have write privileges for a patient', async () => {
-      const { container } = setup({ permissions: [Permissions.ReadPatients] })
-
-      await waitForElementToBeRemoved(container.querySelector(`[class^='css']`))
+      setup({ permissions: [Permissions.ReadPatients] })
 
+      expect(await screen.findByRole('alert')).toBeInTheDocument()
```

Comment:
Probably getting nit-picky on this one. Similar to https://github.com/kcdraidgroup/hospitalrun-frontend/pull/273/files#r557056312 - the semantics of this no longer say "wait for the css spinner to disappear before checking the button isn't there".

As it stands with the alert, I am now led to believe the alert is somehow important to this test.